packages/core/angular/src/connection/connection.types.ts

```diff
@@ -1,5 +1,8 @@
+import { WalletAdapterNetwork } from '@solana/wallet-adapter-base';
 import { Connection } from '@solana/web3.js';
 
 export interface ConnectionState {
     connection: Connection | null;
+    network: WalletAdapterNetwork;
```

Comment:
I don't think the `network` is actually relevant to the Connection object -- the external interface can just be `endpoint`.

Reply:
Indeed, I just wanted to store the network somewhere in the state. I agree that the ConnectionStore is probably not the best place.